colors/nord.vim

```diff
@@ -797,3 +797,56 @@ function! NordPalette() abort
   let ret["nord3_bright"] = s:nord3_gui_bright
   return ret
 endfunction
+
+" Treesitter
+if has("nvim") && exists("g:nord_enable_treesitter")
```

Comment:
Since these are just simple syntax highlighting group definitions there is no need to hide it behind an additional toggle since this won't affect the overall performance and plugin loading time in no way.
Maybe https://github.com/arcticicestudio/nord/issues/157 will be taken into account later on so these definitions might be moved anyway.

```suggestion
if has("nvim")
```